### PR TITLE
CON-2467-track-deeplink-push

### PR DESCRIPTION
**Pull request by Carsten89**

### CODE REVIEW

File: subscriptions/deeplinkPush.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ * Deeplink and push message tracking subscriptions.
+ * @param {Function} subscribe The subscribe function.
+ */
+export default function deeplinkPush(subscribe) {
+  /**
+   * Gets triggered on deeplink opened
+   */
+  subscribe(deeplinkOpened$, ({ action }) => {
+    const { link = '', sourceApp = '', wasOpenedFromSearchIndex } = action.payload;
+
```
Comment (devbucket) [inline subscriptions/deeplinkPush.js]: Remove blank line

File: subscriptions/deeplinkPush.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+const pushOpened$ = main$
+  .filter(({ action }) => action.type === OPEN_PUSH_NOTIFICATION);
+
+/**
+ * Deeplink and push message tracking subscriptions.
+ * @param {Function} subscribe The subscribe function.
+ */
+export default function deeplinkPush(subscribe) {
+  /**
+   * Gets triggered on deeplink opened
```
Comment (devbucket) [inline subscriptions/deeplinkPush.js]: Please re-write this to a full human-readable sentence, please.
Something like `Gets triggered when a deeplink was opened.`

File: subscriptions/deeplinkPush.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    });
+
+    core.track.setCampaignWithUrl({
+      url: link,
+      type: 'deeplink',
+    });
+  });
+
+  /**
+   * Gets triggered on push opened
```
Comment (devbucket) [inline subscriptions/deeplinkPush.js]: Please also re-write to a full human-readable sentence, please.

File: subscriptions/deeplinkPush.js (hunk 1 above)
Comment (Carsten89) [reply subscriptions/deeplinkPush.js]: done

File: subscriptions/deeplinkPush.js (hunk 2 above)
Comment (Carsten89) [reply subscriptions/deeplinkPush.js]: done

File: subscriptions/deeplinkPush.js (hunk 3 above)
Comment (Carsten89) [reply subscriptions/deeplinkPush.js]: done

**Pull request merged**